src/readfish/entry_points/targets.py

```diff
@@ -318,41 +325,48 @@ def check_override_action(
         # If --duplex flag override decisions made based on the strand and contig alignment of the previous read.
         # Unfinished bruv
```

Comment:
Do we need to have this comment now?

Reply:
I need to double check the "more complex" duplex - it should be removed after that!